tests/cross_functional/stress/cephfs/test_breakpoint.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.helpers.cephfs_stress_helpers import create_cephfs_stress_pod
+
+logger = logging.getLogger(__name__)
+
+
+@magenta_squad
+class TestCephfsStress(E2ETest):
+    """
+    CephFS break point test - without failures
+    """
+
+    def test_cephfs_breakpoint(self):
```

Comment:
do we have teardown part implemented ?

Reply:
This is a positive flow and we won't be deleting the created data. Hence, there is no need for teardown code here